docs/guide/extending_form_builder.md

```diff
@@ -0,0 +1,120 @@
+# Extending Form Builder
+
+!!! enterprise
+
+    You can extend the Form Builder by adding new form fields or modifying existing ones.
+    Form fields are defined in YAML configuration.
+    See example of the built-in Single line input:
+
+    ``` yaml
+    ez_platform_form_builder:
+        fields:
+            single_line:
+                name: 'Single line input'
+                category: 'Default'
+                thumbnail: '/bundles/ezplatformadminui/img/ez-icons.svg#input-line'
+                attributes:
+                    label:
+                        name: 'Display label'
+                        type: 'string'
+                        validators:
+                            not_blank:
+                                message: 'You must provide label of the field'
+                    placeholder:
+                        name: 'Placeholder'
+                        type: 'string'
+                    help:
+                        name: 'Help text'
+                        type: 'string'
+                    default_value:
+                        name: 'Default value'
+                        type: 'string'
+                validators:
+                    required: ~
+                    min_length: ~
+                    max_length: ~
+                    regex: ~
+    ```
+
+    Available attribute types are:
+
+    - `string`
+    - `choices`
+    - `text`
+    - `integer`
+    - `url`
+    - `multiple`
+    - `radio`
+    - `action`
+    - `select`
+
+    New types of fields require a mapper. The mapper must be registered as a service:
```

Comment:
Information which interface should be implemented is crucial here:  `\EzSystems\EzPlatformFormBuilder\FieldType\Field\FieldMapperInterface`. 

I think we should provide an example, as this is fundamental extension point :wink: